airbyte-commons/src/test/java/io/airbyte/commons/resources/MoreResourcesTest.java

```diff
@@ -59,10 +61,10 @@ void testListResource() throws IOException {
   }
 
   @Test
-  void testWriteResource() throws IOException {
+  void testWriteToTmpFile() throws IOException {
```

Comment:
This was throwing errors for me when I ran it using Intellij. The classloader was attempting to write to the JUnit Platform jar instead of a proper file location. 

Ideally, I would split this out to a separate PR. However it would involve cleaning this one up, and would affect my development work on this PR. Thus, I'm going to combine these together.